### DISCUSSION TITLE
Prerequisites

### DISCUSSION THREAD

**Discussion by mevdev** [sparkfabrik/ios-build-action#8] [General]
Should a project already have fastlane installed?

I am wanting to get this working and use the artifact with apple-actions/upload-testflight-build. Are there any scripts that interplay more appropriately to do this same thing of signed build + TestFlight upload?

**Comment by edodusi** [comment]
There is a PR that will allow soon to have a signed build and upload it directly to TestFlight: https://github.com/sparkfabrik/ios-build-action/pull/9

You don't have to have Fastlane installed locally, but just have a github repo to store the certificates and an Apple Key to allow the script to connect to Apple Connect.
